### DISCUSSION TITLE
How many servers can I add to teleport free version ?

### DISCUSSION THREAD

**Discussion by Touchmeshuu11** [gravitational/teleport#6725] [Q&A] [answered]
i confused about how many server and trusted cluster can i add to teleport free version. where i can find it or anyone can tell me ?

**Comment by klizhentas** [comment, marked as answer]
Teleport OSS supports 10K connected nodes and 1K connected clusters:

https://goteleport.com/docs/admin-guide/#teleport-scalability-tweaks

**Reply by Touchmeshuu11** [reply]
thank you sir

**Reply by Touchmeshuu11** [reply]
sir can you tell me, how many user can i create with google authenticator on teleport free version ?
